js/ajax.js

```diff
@@ -9,45 +9,35 @@
 
   'use strict';
 
-    // Get our original AJAX behavior so we can still call it.
```

Comment:
This change may be better to go in separate to this? Well need to up our min version of Drupal along side it.

Reply:
Cool, I've already got an MR for that.